PressureSensorAdapter.java

```diff
@@ -0,0 +1,37 @@
+import java.util.Random;

```

Comment:
The adapter classes are redundant from rewrite each methods from Sensor class. Improve example:
@Override
    public String getName() {
            return adaptee.getName();
}